projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -69,7 +69,7 @@ export class IgxGridNavigationService {
     public dispatchEvent(event: KeyboardEvent) {
         const key = event.key.toLowerCase();
         if (!this.activeNode || !(SUPPORTED_KEYS.has(key) || (key === 'tab' && this.grid.crudService.cell)) &&
-            !this.grid.crudService.rowEditingBlocked && !this.grid.crudService.rowInEditMode) {
+            !this.grid.crudService.rowEditingBlocked && !!this.grid.crudService.rowInEditMode) {
```

Comment:
we have the same check and inside hierarchical-grid-navigation.service ... please address the change correspondingly

Reply:
Sigh, itch to completely rework all of this intensifies. But will try to resists for the fix PRs :D